### DISCUSSION TITLE
Release v1.11.4 / 5.66.4

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3365] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes
This is a maintenance release fixing a various issues and adds minor improvements

For a full list of changes and fixes please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1114--5664---2023-10-06).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.11.4'>Release v1.11.4 / 5.66.4</a>.</em>

**Comment by BilleBarrett** [comment]
Since Sandboxie Plus updated I'm getting this:
chrome.exe: SBIE2321 Cannot manage device map:  [C0000034 / 11]
how do I fix it, I also see it on other computers.
Thank you
Bill e. Barrett

**Reply by Sekt0rX** [reply]
Just confirming this is still an issue as of v1.12.7. 
Assume it's still due to running as Admin + DropAdminRights=y, as mentioned in https://github.com/sandboxie-plus/Sandboxie/issues/2996

I haven't updated in awhile but I didn't have this error appear when I was running v1.5.3.
Thanks.

**Reply by isaak654** [reply]
I notified @DavidXanatos via Slack a few months ago, but was asked to provide reproducible / fool-proof steps. Since this didn't occur so far, please do so in that issue. Alternatively, I could send an invitation to join the Slack channel if you are confident to provide an email address that can be deleted later.

**Reply by Sekt0rX** [reply]
I'm not sure exactly what steps I can provide, I'd just be repeating what's already been mentioned in that ticket. 
I receive the error any time I run any program in Sandboxie since I updated from 1.5.3 to 1.11.something, I then updated to 1.12.7 today and still receive the message. I've never seen the message on 1.5.3 (or the versions I've run before that).

I launch the programs using the context menu "Run Sandboxed" or the Program Control/Force Programs option in Sandboxie, it doesn't matter what program, the error message is displayed (for example, Notepad++, Brave/Chrome Browser, just tried Microsoft Paint).
I run Windows 10 Pro 22H2 (19045.3570) and the Windows account type I use is Local User Account (Administrator).

Unticking "Drop rights from Administrators and Power User groups" in the Sandboxie options stops the error from appearing.

**Reply by isaak654** [reply]
> I'm not sure exactly what steps I can provide, I'd just be repeating what's already been mentioned in that ticket.

When steps are required, the developer wants the issue to be presented in a more compact and comprehensible way. 
For example, see https://github.com/sandboxie-plus/Sandboxie/issues/2025#issuecomment-1200110235 and https://github.com/sandboxie-plus/Sandboxie/issues/3537#issuecomment-1880165636

It is unfortunate that there are issues that will not be fixed because of this, but no other developers are currently available.

**Reply by isaak654** [reply]
Update: https://github.com/sandboxie-plus/Sandboxie/issues/2996#issuecomment-1913284360
